tests/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ TEST(TestDiveSiteDuplication testdivesiteduplication.cpp)
 TEST(TestRenumber testrenumber.cpp)
 TEST(TestGitStorage testgitstorage.cpp)
 TEST(TestPreferences testpreferences.cpp)
+TEST(TestPicture testpicture.cpp)
```

Comment:
you add that file in the next commit. That could be painful when bisecting - can you move the change to CMakeLists.txt into the second commit, please?